### ISSUE TITLE
Commodity price calculations incorrect for technologies with output quantities not equal to 1

### ISSUE THREAD

**Issue by tsmbland** [EnergySystemsModellingLab/MUSE_OS#552]
Commodity prices are calculated using the `annual_levelized_cost_of_energy` function, which calculates the cost of operating one unit of a technology at full capacity. However, it doesn't take output quantities into account, essentially assuming that one unit of capacity produces one unit of output. But this may be (very) incorrect. For a output quantity x, commodity prices will end up being x-fold higher than they should be.

Basically this function needs a complete rewrite to be more like the `lifetime_levelized_cost_of_energy` function:
- take capacity into account
- use exponents
- take output quantities into account

Also, see #551
